CODEOWNERS

```diff
@@ -20,6 +20,7 @@
 /build-tools/manifest-attribute-codegen @jonpryor
 /build-tools/scripts/PrepareWindows.targets @jonathanpeppers
 /build-tools/timing @jonathanpeppers @radekdoulik
+/build-tools/xabootstrap @grendello
```

Comment:
Minor nitpick: should this be `bootstrap` or `prepare`?  It's not like we're going to have multiple bootstrap systems in here...